mlrun/serving/server.py

```diff
@@ -544,11 +544,18 @@ def __init__(
         self.get_store_resource = None
         self.get_table = None
         self.is_mock = False
+        self._project_obj = None
 
     @property
     def server(self):
         return self._server
 
+    @property
+    def project_obj(self):
+        if not self._project_obj:
+            self._project_obj = mlrun.load_project(url=self.project, save=False)
```

Comment:
load_project does more than just loading the project instance for project operations. e.g `project_setup.py` would be executed part of loading the project.
why not simply using httpdb to get the project and you will have the project instance at hand for project operations (like get function, store artifact, etc).